lib/xboxrt/libc_extensions/locale_ext_.h

```diff
@@ -0,0 +1,27 @@
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// We don't really have locale support.
+// These are only stubs, providing just the bare minimum for libc++.
+
+typedef void *_locale_t;
+
+static _locale_t _create_locale (int category, const char *locale)
```

Comment:
These functions (and type?) should be marked as Windows specific stuff. If we also add some POSIX extensions later we don't want to lose track.

Reply:
The comment above it now mentions that it's Windows-specific.